README.md

```diff
@@ -20,15 +20,190 @@ tasks specified in the Gruntfile.
+```
+$ echo ~
+```
+in a **terminal**. ('~' is a shortcut for the home directory.) Macs use a hidden file called `.zsh` in the home directory to set variables like *PATH*. Other Unix systems use files like `.bashrc`. You might have to check the specifics for your distribution to know which you should use. Open the appropriate file. If the file does not already exist, create a new file that will be saved with the appropriate name in your home directory.
+We want to add the location of the **adb** tool to your *PATH* while preserving the existing *PATH* information. Assuming that your **adb** program is in the `/Users/someuser/Desktop/android-sdk/platform-tools/` directory, you would add the following command to the end of the `.zshrc` file:
+```
+$ export PATH=${PATH}:/Users/someuser/Desktop/android-sdk/platform-tools
+```
+Save the file, close the **terminal** window, open a new **terminal** window, and type:
+```
+$ echo $PATH
+```
+You should see your old path with the new directory you added above, and you should now be able to run:
+```
+$ adb --version
+```
+
+**Tip:** If you are going to be heavily customizing the look-and-feel of the application with a lot of external JavaScript libraries, you might also choose to install **bower**.
+
+## Installing Application Designer
+Open a **Terminal** and type
+```
+$ git clone https://github.com/odk-x/app-designer.git
+```
+If you're using **Github Desktop**, clone [this repository](https://github.com/odk-x/app-designer) by clicking on `Code`.
+
+Install the [grunt packages](https://github.com/iamhks/app-designer#grunt) also mentioned in the prerequisites. Make sure [NodeJS](https://github.com/iamhks/app-designer#nodejs) is also installed.
+
+To open Application Designer, navigate to the location of your local cloned repository in **cmd**/**Terminal**, and type: 
+```
+$ grunt
+```
+This command runs the script contained in **Gruntfile.js**, so be sure it is in the current directory.
+
+#### Windows Users Tip
+You will be opening a **cmd** window and changing your current directory (using the **cd** command) into this directory every time you use this tool.  It is therefore useful to create a shortcut that opens a cmd window directly into this directory:
+1) Open a file browser and navigate to the unzipped directory containing a number of files and directories, including a **Gruntfile.js**.
+2) Click into the top location bar that displays the nested list of folders to this folder.
+3) Copy this path to the cut-and-paste buffer.
+4) Now, move down to the list of files, right-click.
+5) Select *New...,Shortcut*.
+6) Type **cmd** for the location of the item.
+7) Click Next, and then Finish.
+8) Select this newly-created **cmd.exe** shortcut and right-click.
+9) Select *Properties*.
+10) Click on the *Start* in text box, delete its contents, and paste the path to this folder.
```

Comment:
The label for this text box is `Start in` not `Start`.

Reply:
Noted and updated.